helm/ph-ee-engine/connector-channel/templates/deployment.yaml

```diff
@@ -80,16 +80,26 @@ spec:
               value: "{{ .Values.tenantSecondary.clientSecret }}"
             - name: "CHANNEL_TENANTSECONDARY_TENANT"
               value: "{{ .Values.tenantSecondary.tenant }}"
+            - name: "server_ssl_key-alias"
+              value: "{{ .Values.server.ssl.key_alias }}"
+            - name: "server_ssl_key-store"
+              value: "{{ .Values.server.ssl.key_store }}"
+            - name: "server_ssl_key-store-type"
+              value: "{{ .Values.server.ssl.key_store_type }}"
+            - name: "server_ssl_key-password"
+              value: "{{ .Values.server.ssl.key_password }}"
+            - name: "server_ssl_key-store-password"
+              value: "{{ .Values.server.ssl.key_store_password }}"
             - name: "redis_idempotency_enabled"
-              value: "{{ .Values.channel.redis.idempotency.enabled }}"
+              value: "{{ .Values.redis.idempotency.enabled }}"
```

Comment:
in the latest HELM Gold standards applied to the template, we don't have to mention the `channel` specifically.

Reply:
It's already included